presto-native-execution/presto_cpp/main/PeriodicTaskManager.cpp

```diff
@@ -108,7 +108,10 @@ PeriodicTaskManager::PeriodicTaskManager(
       asyncDataCache_(asyncDataCache),
       arbitrator_(velox::memory::memoryManager()->arbitrator()),
       connectors_(connectors),
-      server_(server) {}
+      server_(server),
+      numStuckDriversToDetachWorker(
```

Comment:
Why use hw multiplier for this?

Reply:
Not too large number and not too big. Also this would tie the threshold to the number of driver threads.
We could use any % - I'm open for other ideas.
As I mentioned in the description, this is not a bullet proof strategy, but  better than the one we have now.